scripts/xnat/check_valid_sessions.py

```diff
@@ -13,19 +13,43 @@
 
 Example
 =======
-- Update the cache and generate the baseline report
-  ./check_valid_sessions --update --baseline
+- When running for the first time run 
+  ./check_valid_sessions.py --update 
+  so that the cach (located at experimentsdir) is created 
+
+- Update the cache (stored in experimentsdir) and generate the baseline report
+  ./check_valid_sessions.py --update --baseline
 
 - Use the existing cache to extract 10 in the followup window
- ./check_valid_sessions --num_extract 10 --min 180 --max 540
+ ./check_valid_sessions.py --num_extract 10 --min 180 --max 540
+
+If you have problems running it log in as nicholsn and run 
```

Comment:
our public code should not have references to loging in under my user name, pls remove